jlm/hls/backend/rhls2firrtl/verilator-harness-hls.cpp

```diff
@@ -1,743 +1,645 @@
 /*
  * Copyright 2021 David Metz <david.c.metz@ntnu.no>
+ * Copyright 2024 Håvard Krogstie <krogstie.havard@gmail.com>
  * See COPYING for terms of redistribution.
  */
 
 #include <jlm/hls/backend/rhls2firrtl/verilator-harness-hls.hpp>
 #include <jlm/llvm/ir/operators/delta.hpp>
 
+#include <sstream>
+
 namespace jlm::hls
 {
 
+// The number of cycles before a load is ready
+static constexpr int MEMORY_RESPONSE_LATENCY = 10;
+
 std::string
-VerilatorHarnessHLS::get_text(llvm::RvsdgModule & rm)
+convert_to_c_type(const rvsdg::Type * type)
```

Comment:
New functions should follow the new naming convention.
ConvertToCType()

Reply:
Changed